Carthage/Checkouts/Abstract/RATIONALE.md

```diff
@@ -0,0 +1,111 @@
+# Rationale
```

Comment:
We shouldn't commit the `Carthage/Checkouts` folder. Please remove from git and add to `.gitignore`.

Reply:
@broomburgo ignored `Carthage/Checkouts` and deleted the folder